Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockSupport.cs

```diff
@@ -65,8 +66,9 @@ public static void Initialize()
             InputSystem.RegisterLayoutMatcher<DualShock3GamepadHID>(
                 new InputDeviceMatcher()
                     .WithInterface("HID")
-                    .WithManufacturer("Sony.+Entertainment")
-                    .WithProduct("PLAYSTATION(R)3 Controller"));
+                    .WithManufacturerContains("Sony")
```

Comment:
I'm wondering it this could be just WithManufacturer("Sony", supportRegex:false) ?

There is a small chance this matches wider now - E.g. Entertainment Sony would match now, but I don't think this is likely to be any false positives.

Reply:
I believe it needs to be an exact match - so would need to provide the entire manufacturer string.

There seem to be two flavours of Sony manufacturer string:

- "Sony Interactive Entertainment"
- "Sony Computer Entertainment"

... hence the regex in the original code.

I can check if this is the case though.